src/main/java/lotto/common/Constants.java

```diff
@@ -0,0 +1,19 @@
+package lotto.common;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Constants {
+
+    public static final int MIN_RANGE_VALUE = 1;
+    public static final int MAX_RANGE_VALUE = 45;
+    public static final int GET_NUMBER_COUNT = 6;
+    public static final int GAME_PRICE = 1000;
+    public static final Map<Integer, Integer> PRIZE_MONEY = new HashMap<Integer, Integer>(){{
+        put(3, 5000);
+        put(4, 50000);
+        put(5, 1500000);
+        put(6, 2000000000);
```

Comment:
```suggestion
        put(6, 2_000_000_000);
```
언더바(_)를 사용해서 숫자의 단위 가독성을 더 좋게 표현할 수 있어요 :)

Reply:
넵 언더바를 사용하는게 가독성이 좋네요.ㅎㅎㅎ
놓쳤던 부분에 대한 피드백 고맙습니다.